hugegraph-api/pom.xml

```diff
@@ -153,6 +153,17 @@
             <artifactId>swagger-jaxrs2-jakarta</artifactId>
             <version>2.1.9</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.taobao.arthas</groupId>
+            <artifactId>arthas-agent-attach</artifactId>
+            <version>3.6.9</version>
```

Comment:
1. unify the version with a param like `${arthas.version}`
2. consider use [3.7.x](https://mvnrepository.com/artifact/com.taobao.arthas/arthas-agent-attach/3.7.1) after basic test

Reply:
> for 2.
> 
> i remember, some days ago, maybe one comment say , we need use stable version,not latest? [alibaba/arthas/releases](https://github.com/alibaba/arthas/releases)

Yep, I mean don't use the code from **master** or `preview-release` or `just released few days (<30d)`
It may not be very clear, and in addition, some projects will maintain a separate **stable version**, while others will only distinguish between **preview** and **official(latest)** release

<img width="1103" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/c48e5292-0e7d-41d7-8dfb-0a5bcc0b9c9c">

here we could try `3.7.1`, thanks for point this